node/test/as-http.js

```diff
@@ -189,7 +189,14 @@ function allocHTTPBridge(opts) {
     }
 
     function onEgressRequest(hreq, hres) {
-        cluster.asHTTP.forwardToTChannel(cluster.egressChan, hreq, hres);
+        var options = cluster.egressChan.requestOptions({
```

Comment:
no, this logic should be put into `asHTTP.forwardToTChannel` itself since this logic will also be needed for real.